src/components/templates/PosterPage/PosterPage.tsx

```diff
@@ -53,19 +52,19 @@ export const PosterPage: React.FC<PosterPageProps> = ({ venue }) => {
 
     becomePassiveParticipant,
     becomeActiveParticipant,
+    localParticipant,
   } = usePosterVideo(venueId);
 
   const videoParticipants = useMemo(
     () =>
       activeParticipants.map(({ participant, user }, index) => (
-        <VideoParticipant
-          key={participant.identity ?? `participant-${index}`}
+        <VideoCommsParticipant
+          key={participant.sparkleId ?? `participant-${index}`}
           participant={participant}
-          participantUser={user}
-          containerClassName="PosterPage__video-participant"
+          isLocal={participant.id === localParticipant?.id}
```

Comment:
[OOS]
If this kind of check should be done regularly and in many places,
looks like it would be useful to put the logic inside a utility function.